coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -127,6 +148,37 @@ def metadata(self):
         """
         return self._metadata
 
+    @property
+    def class_padding(self):
+        """
+        A namedtuple consisting of values about blank lines before and after
+        the documentation of ``docstring_type`` class.
+
+        These values are official standard of following blank lines before and
+        the documentation of ``docstring_type`` class.
+        """
+        return self._class_padding
+
+    @property
+    def function_padding(self):
+        """
+        A namedtuple consisting of values about blank lines before and after
+        the documentation of ``docstring_type`` function.
+
+        These values are official standard of following blank lines before and
+        the documentation of ``docstring_type`` function.
+        """
+        return self._function_padding
+
+    @property
+    def docstring_type_regex(self):
```

Comment:
hm, ok... since Python doesn't have private members, the whole concept is a bit useless. But if you want to do it like that it's fine.